src/frontend/web_application/src/store/modules/user.js

```diff
@@ -39,6 +41,19 @@ export function updateUser({ user, original }) {
   };
 }
 
+export function deleteUser({ user, password }) {
+  return {
+    type: DELETE_USER,
+    payload: {
+      request: {
+        method: 'post',
+        url: `/api/v2/${user.user_id}/actions`,
+        data: { password },
```

Comment:
^^ you forgot to push your latest changes @AlxxxlA. Are you able to do it soon ?